CHANGELOG.md

```diff
@@ -0,0 +1,82 @@
+# Changelog
+
+Notable changes to the project will be documented in this file.
+
+## [6.0.0](https://github.com/kirbydesign/designsystem/pull/2337) (15-06-2022)
+
+### Bugfixes
+
+**button** - change size of icon on icon-only buttons (#2306)
+**form-field** - medium sized date type inputfield layout error (#2314)
```

Comment:
All the lists of features and fixes in this file lack a newline at the end:  https://github.com/kirbydesign/designsystem/blob/documentation/remove-component-status/CHANGELOG.md

Could we put them in an unordered list?
Perhaps also convert each PR reference to a link? (Quite a task - sorry if you end up doing it! 😬 )

Reply:
Fixed it!